internal/oidc/keyhandler.go

```diff
@@ -3,6 +3,7 @@ package oidc
 import (
 	"context"
 	"fmt"
+	"github.com/lestrrat-go/jwx/jwa"
 	"net/http"
 	"sync"
 	"time"
```

Comment:
Place the non-std lib import as follows:

```suggestion
	"context"
	"fmt"
	"net/http"
	"sync"
	"time"
	
	"github.com/lestrrat-go/jwx/jwa"
```